packages/FormElement/README.md

```diff
@@ -22,107 +22,172 @@ npm install @paprika/form-element
+| children | node | false    | null    | Content for right column flex child (typically everything except the Label) |
+
+### Fieldset
+
+Props are the same as `FormElement` but `hasFieldSet=true`.
+
 ## Usage
 
-Please use `<L10n />` component to wrap `<FormElement />` or you application.
+The `<L10n />` component is a required context provider that must wrap the `<FormElement />` (or your application) for proper localization.
 
-Using with paprika `<Input />`
+### Using with `<Input />`
 
+<!-- prettier-ignore -->
 ```jsx
-import FormElement from "@paprika/form-element";
 import Input from "@paprika/input";
+import FormElement from "@paprika/form-element";
+
+const { Label, Content } = FormElement;
 
 <FormElement>
-  <Label>Name</Label>
+  <Label>First Name</Label>
   <Content>
     {a11yProps => (
       <Input
         onChange={handleChange}
         value={value}
-        placeholder="Form placeholder"
-        aria-required={hasRequiredLabel}
-        hasError={false}
-        isDisabled={isDisabled}
-        isReadOnly={isReadOnly}
-        size={size}
         {...a11yProps}
       />
     )}
-    )}
   </Content>
-</FormElement>;
+</FormElement>
 ```
 
-Using with paprika `<DatePicker />`
+### Using with `help` popover, `<Instructions />`, `<Description />`, and `<Error />`
+
+Best practices for laying out supporting content for a form input is to follow this order:
 
+- `<Label />` first.
+- `<Instructions />` above the input field contained in `<Content />`.
+- `<Description />` or `<Error />` shown below input field (but do not render both at the same time).
+
+Note: the `<Error />` component will render `null` if its `children` is falsey.
+
+<!-- prettier-ignore -->
 ```jsx
-import DatePicker from "@paprika/date-picker";
-import FormElement, { Label, Content } from "@paprika/form-element";
+import Input from "@paprika/input";
+import FormElement from "@paprika/form-element";
+
+const { Label, Content, Instructions, Description, Error } = FormElement;
+const errorMsg = "There was an error.";
 
 <FormElement>
-  <Label>Name</Label>
+  <Label help="Help text">Field label</Label>
+  <Instructions>Instruction text</Instructions>
+  <Content>
+    {a11yProps => (
+      <Input hasError={Boolean(errorMsg)} {...a11yProps} />
+    )}
+  </Content>
+  {errorMsg ? <Description>Description text</Description> : null}
+  <Error>{errorMsg}</Error>
+</FormElement>
+```
+
+### Using `<Fieldset />` with `<Checkbox />`
+
+Since the `<Checkbox />` and `<Radio />` components already render with a `<label>` element, the label for the checkbox or radio _group_ needs to be a `<legend>` element, instead of another redundant `<label>`, which needs to be wrapped in a `<fieldset>` element. For this purpose, the `<FormElement />` has a `hasFieldSet` prop, but for clearer JSX markup, you can use the `<Fieldset>` component, provided as a named import from the same `@paprika/form-element` component.
```

Comment:
Yeah, probably a good suggestion.  Easier to write this stuff in `md` than `html`, but it would be good content to have in that story.  I guess I don't mind having it in both places, but maybe I can trim the detail from the README, but keep it all in the story.